graph_service_client.go

```diff
@@ -58,3 +61,13 @@ func (m *GraphBaseServiceClient) GetAdapter() abstractions.RequestAdapter {
 	}
 	return m.requestAdapter
 }
+
+// Me provides operations to manage the user singleton.
+func (m *GraphBaseServiceClient) Me() *if6ffd1464db2d9c22e351b03e4c00ebd24a5353cd70ffb7f56cfad1c3ceec329.UserItemRequestBuilder {
+	urlTplParams := make(map[string]string)
+	for idx, item := range m.pathParameters {
+		urlTplParams[idx] = item
+	}
+	urlTplParams["user%2Did"] = "me-token-to-replace"
+	return if6ffd1464db2d9c22e351b03e4c00ebd24a5353cd70ffb7f56cfad1c3ceec329.NewUserItemRequestBuilderInternal(urlTplParams, m.requestAdapter)
+}
```

Comment:
Pciked changes from https://github.com/microsoftgraph/msgraph-sdk-go/issues/347

Reply:
are we planning to merge this PR since it has the UUI serialization bug? shouldn't we regenerate?

Reply:
I meant to type UUID